version.properties

```diff
@@ -1,4 +1,3 @@
-#Version of the produced binaries. This file is intended to be checked-in.
-#It will be automatically bumped by release automation.
-version=1.3.1
-previousVersion=1.3.0
+# Version of produced binaries.
+# The '*' part is automatically resolved by shipkit-auto-version Gradle plugin (https://github.com/shipkit/shipkit-auto-version).
+version=1.3.*
```

Comment:
Does this cover Pioneers versioning scheme (bump minor for feature release, bump patch for hotfix release)?

Reply:
Yes. Patch version is incremented automatically with every release. To increment major or minor version one needs to change the `version.properites` file and change `version=1.3.*` to `version=1.4.*` or `version=2.0.*`.